spec/support/capybara.rb

```diff
@@ -11,6 +11,9 @@
 Capybara.default_driver = :rack_test
 Capybara.javascript_driver = :chrome
 
+# change :selenium_chrome_headless to just :selenium in this file in order to see your tests and troubleshoot in browser
```

Comment:
Maybe we should try to set this up as an argument that gets passed in so that we can do something like `rspec -headless` in the travis file and locally. That way it does the browser by default. That's not a huge priority though

Reply:
Good idea, Brian.